content/en/apps/guides/data/analytics/couch2pg-to-cht-sync-migration.md

```diff
@@ -0,0 +1,28 @@
+
+## Key Considerations
+- **Server resources**: To minimize downtime, it is recommended to run both Couch2pg and CHT Sync in parallel during the migration process. With this in mind, ensure that the server and database resources are sufficient to handle the load.
+- **DBT Modelling**: Avoid the temptation to model new DBT models after existing SQL views and tables. Instead, take the opportunity to re-evaluate the data needs and design new models that are more efficient and effective. Think of what data needs to be shown and how it should be shown and use that to guide the design of the new models.
+- **Testing**: After migrating, thoroughly test the new DBT models to ensure that they are working as expected.
+- **Feedback**: Provide any feedback and create issues for errors or bugs encountered in the [CHT Sync](https://github.com/medic/cht-sync) and [CHT Pipeline](https://github.com/medic/cht-pipeline/) repos to make it easier for the next migration.
+
+## Migration Steps
+1. **Plan Migration**: Determine the scope of the migration, including the data sources, the data models, and the data transformations. Identify the SQL views and tables that need to be converted into DBT models.
+1. **Set up CHT Sync**: Follow the instructions in the [Local CHT Sync Setup](({{< relref "apps/guides/data/analytics/setup" >}})) guide to set up CHT Sync locally and start DBT modelling.
+1. **Deploy CHT Sync**: Once the DBT models are tested locally and working as expected, deploy CHT Sync in production. Follow the instructions in the [Production CHT Sync Setup](({{< relref "apps/guides/data/analytics/production" >}})) guide to set up CHT Sync in production. Note that we recommend running CHT Sync in parallel with Couch2pg during the migration process.
+1. **Test New Models**: After deploying CHT Sync, test the new DBT models to ensure that they are working as expected. Make any necessary adjustments to the models to ensure that they are accurate and efficient. To do this you will need to create a replica dashboard and compare the data from the old and new pipelines.
+1. **Optimize**: Once the new DBT models are working as expected, optimize the models to improve performance. This may involve restructuring the models, adding indexes, or making other adjustments to improve the speed and efficiency of the models. Having a look at the [DBT Models](({{< relref "apps/guides/data/analytics/building-dbt-models" >}})) guide will help you understand how to optimize the models.
```

Comment:
```suggestion
1. **Optimize**: Once the DBT models are working as expected and the dashboards display the expected data, optimize the models to improve performance. This may involve restructuring the models, adding indexes, or making other adjustments to improve the speed and efficiency of the models. Having a look at the [DBT Models](({{< relref "apps/guides/data/analytics/building-dbt-models" >}})) guide will help you understand how to optimize the models.
```